modules/cron4s/src/main/scala/pureconfig/module/cron4s/package.scala

```diff
@@ -2,14 +2,17 @@ package pureconfig.module
 
 import _root_.cron4s.expr.CronExpr
 import _root_.cron4s.Cron
-import pureconfig.ConfigReader
+import pureconfig.{ ConfigReader, ConfigWriter }
 import pureconfig.error.CannotConvert
 
 package object cron4s {
 
-  implicit val cronExprReader: ConfigReader[CronExpr] =
+  implicit val cronExprConfigReader: ConfigReader[CronExpr] =
     ConfigReader.fromString(str =>
       Cron.parse(str).fold(
         err => Left(CannotConvert(str, "CronExpr", err.getMessage)),
         cronExpr => Right(cronExpr)))
+
+  implicit val cronExprConfigWriter: ConfigWriter[CronExpr] =
```

Comment:
Can you combine the reader and writer parts into a `ConfigConvert`? You should be able to do that easily with `ConfigConvert.viaString` (see [pureconfig-joda](https://github.com/pureconfig/pureconfig/blob/master/modules/joda/src/main/scala/pureconfig/module/joda/package.scala) for examples).

Reply:
@ruippeixotog thanks to the pointer to `ConfigConvert`, wasn't aware of its existance. Should be sorted now.